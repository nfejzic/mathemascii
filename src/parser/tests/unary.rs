use super::Snapshot;

super::test_snap!(accents, "hat x vec x tilde x");
super::test_snap!(others, "sqrt x text(some text)");
super::test_snap!(groupings, "abs(x) floor(x) norm x");
super::test_snap!(font_commands, "fr text(other text)");
