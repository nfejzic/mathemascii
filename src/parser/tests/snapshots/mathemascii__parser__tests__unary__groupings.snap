---
source: src/parser/tests/unary.rs
expression: "Snapshot((input, math))"
---
abs(x) floor(x) norm x

Interm {
  IntermExpr {
    Absolute(
      OpenParen
        Interm {
          IntermExpr {
            Variable("x")
          }
        }
      CloseParen
    )
  }
}

Interm {
  IntermExpr {
    Floor(
      OpenParen
        Interm {
          IntermExpr {
            Variable("x")
          }
        }
      CloseParen
    )
  }
}

Interm {
  IntermExpr {
    Norm(
      Variable("x")
    )
  }
}


